packages-exp/auth-exp/src/core/util/emulator.ts

```diff
@@ -20,13 +20,12 @@ import { debugAssert } from './assert';
 
 export function _emulatorUrl(config: ConfigInternal, path?: string): string {
   debugAssert(config.emulator, 'Emulator should always be set here');
-  const { hostname, port } = config.emulator;
+  const { host: emulatorHost } = config.emulator;
```

Comment:
nit: we should probably do some sort of URI.parse() here to standardize the URL

Reply:
Done